main.py

```diff
@@ -101,11 +84,71 @@ def process_data():
     print(df4.to_string())
     df4.to_csv("filtered_data.csv")
 
+def cal_dividend_increament(div_obj: 'pandas.core.series.Series', number_of_years: 'int')-> dict:
```

Comment:
in the annotations it should be the type without ' ':
def cal_dividend_increament(div_obj: pd.core.series.Series, number_of_years : int) -> int